ion-java-cli/pom.xml

```diff
@@ -0,0 +1,121 @@
+<!--
+  ~ Copyright 2007 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.amazon.ion</groupId>
+  <artifactId>ion-java-cli</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>
+    A Java CLI used for ion-test-driver
+  </description>
+  <url>https://github.com/amzn/ion-java/</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Amazon Ion Team</name>
+      <email>ion-team@amazon.com</email>
+      <organization>Amazon</organization>
+      <organizationUrl>https://github.com/amzn</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:amzn/ion-java.git</connection>
+    <developerConnection>scm:git:git@github.com:amzn/ion-java.git</developerConnection>
+    <url>git@github.com:amzn/ion-java.git</url>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.build.timestamp.format>yyyy</maven.build.timestamp.format>
+    <build.year>${maven.build.timestamp}</build.year>
+    <jdkVersion>1.8</jdkVersion>
+  </properties>
+
+  <dependencies>
+    <!-- ion-java/tools/cli dependencies -->
+    <dependency>
+      <groupId>args4j</groupId>
+      <artifactId>args4j</artifactId>
+      <version>2.33</version>
+    </dependency>
+    <dependency>
+      <groupId>com.amazon.ion</groupId>
+      <artifactId>ion-java</artifactId>
+      <version>1.7.0</version>
```

Comment:
We might want this to be open-ended so that the newest ion-java version available will be used. The syntax for this would be `[1.7.0,)`.

Reply:
Changed.